src/ble.rs

```diff
@@ -0,0 +1,179 @@
+//! Connect to an LED badge via Bluetooth Low Energy (BLE)
+
+use std::time::Duration;
+
+use anyhow::{Context, Result};
+use btleplug::{
+    api::{Central as _, Manager as _, Peripheral as _, ScanFilter, WriteType},
+    platform::{Manager, Peripheral},
+};
+use tokio::time;
+use uuid::Uuid;
+
+use crate::protocol::PayloadBuffer;
+
+/// `0000fee0-0000-1000-8000-00805f9b34fb`
+const BADGE_SERVICE_UUID: Uuid = btleplug::api::bleuuid::uuid_from_u16(0xfee0);
+/// `0000fee1-0000-1000-8000-00805f9b34fb`
+const BADGE_CHAR_UUID: Uuid = btleplug::api::bleuuid::uuid_from_u16(0xfee1);
```

Comment:
I guess that's what I wrote in my suggestion, but it would be a bit nicer to read with an `use btleplug::api::bleuuid` statement 🙈:
```suggestion
const BADGE_CHAR_UUID: Uuid = bleuuid::uuid_from_u16(0xfee1);
```